src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqModelRepository.kt

```diff
@@ -23,6 +23,7 @@ class JooqModelRepository(dsl: DSLContext) : JooqRepository(dsl), ModelRepositor
                 GENDER.CODE.`as`("gender"),
                 MODEL.CURRENT_VERSION)
                 .fromJoinPath(MODEL, GENDER, joinType = JoinType.LEFT_OUTER_JOIN)
+                .where(MODEL.IS_CURRENT.eq(true))
```

Comment:
Putting this here also means you'll get an error if you ask for a specific model which is not current

Reply:
Cool, we can always adapt this in the future if needed :+1: